lutris/util/timer.py

```diff
@@ -2,20 +2,30 @@
 
 
 class Timer:
+    """Simple Timer class to time code.
+    You can overload the + operator according to your need by using a new timer_type.
+    """
+
+    def __init__(self, timer_type=None):
+        self.timer_type = timer_type
+        self.finsihed = False
 
     def start_t(self):
-        self.start = datetime.datetime.now()
+        self._start = datetime.datetime.now()
 
     def end_t(self):
-        self.end = datetime.datetime.now()
+        self._end = datetime.datetime.now()
+        self.finsihed = True
 
     def duration(self):
```

Comment:
with property a user could access the duration as timer.duration and the class would be doing the work behind the scenes to convert it to seconds/hours/minutes/days or whatever is specified in self.format. Additionally when self._end hasn't been set, return current duration.